### PR TITLE
Profile heading

### PR DESCRIPTION
**Pull request by CalvinFronda**
Updated Profile heading

TODO:
 Tests - Need write tests for logging into different accounts and checking profiles with search.

### CODE REVIEW

File: src/components/FeatureViews/Accounts/ProfileHeading.js [hunk 1]
```diff
@@ -124,14 +223,14 @@ const ProfileHeading = ({ user }) => {
 							<GradeIcon className="iconScaling" />
 						</Grid>
 						<Grid className="rowPadding" item xs={5}>
-							<Typography className="rowText">Grade {user.grade}</Typography>
+							<Typography className="rowText">Grade {userInfo.grade}</Typography>
```
Comment (a0u5) [inline src/components/FeatureViews/Accounts/ProfileHeading.js:226]: add ?

Comment (jsflca2012) [inline src/components/FeatureViews/Accounts/ProfileHeading.js:226]: add ?

File: src/components/FeatureViews/Accounts/TabComponents/ParentContact.js [hunk 2]
```diff
@@ -9,42 +9,61 @@ import gql from "graphql-tag";
...
+	const {data,loading, error} = useQuery(GET_PARENTS, {
+		variables: {userID :accountID}
+	})
+		
 	if (loading) return <Loading loadingText="PARENT LOADING" small />;
 	if (error) return <LoadingError error="parent" />;
 
+	// Needs to take ID and check if its a parent
+	const {student} = data;
+	console.log(student)
```
Comment (jsflca2012) [inline src/components/FeatureViews/Accounts/TabComponents/ParentContact.js:51]: remove console.log

**Review by jsflca2012 — COMMENTED** [review]
looks good to me

**Pull request merged**